core/src/main/java/org/apache/iceberg/BaseFile.java

```diff
@@ -155,7 +153,7 @@ public PartitionData copy() {
   /**
    * Copy constructor.
    *
-   * @param toCopy a generic data file to copy.
+   * @param toCopy   a generic data file to copy.
```

Comment:
Nit: unnecessary whitespace changes can cause commit conflicts.

Reply:
Okay, will revert this change.